app/backend/templates/v2/event_delete.mjml

```diff
@@ -0,0 +1,15 @@
+<mj-section padding="0px">
+  <mj-column>
+    <mj-text>Hi {{ user.name|v2esc }},</mj-text>
+    <mj-text>An event you are subscribed to has been deleted by the moderators:</mj-text>
+    <mj-text><b>{{ event.title|v2esc }}</b><br />
+      {{ time_display|v2esc }}<br />
+      <i>{% if event.offline_information.address %}{{ event.offline_information.address }}{% endif %}</i>
```

Comment:
This should be the full thing I guess?

Reply:
As a deleted event cannot be accessed anymore, I thought it made sense to still show the address, but not have any links anymore. Or am I seeing this wrong?

Reply:
Ahh, hmm. I think we can either have all the info (incl. link) or then forgo it entirely.

This will only be sent to those who already subscribed, so I don't think it's much of an issue to keep the info there.

Reply:
I think we would want the info to not be shown because if an event needs deleting, we wouldn't want any users (even those subscribed) to be able to see any information beyond the event title (so they know which event was deleted).